projects/skia/build.sh

```diff
@@ -43,7 +43,9 @@ $SRC/depot_tools/gn gen out/Fuzz\
     skia_enable_gpu=false
     extra_ldflags=["-lFuzzingEngine", "'"$CXXFLAGS_ARR"'"]'
 
-$SRC/depot_tools/ninja -C out/Fuzz_mem_constraints image_filter_deserialize textblob_deserialize
+$SRC/depot_tools/ninja -C out/Fuzz_mem_constraints image_filter_deserialize \
```

Comment:
Why not just cp image_filter_deserialize twice?  Once to $OUT/image_filter_deserialize and once to $OUT/image_filter_deserialize_width

Reply:
Done